lib/useKWindowDimensions.js

```diff
@@ -0,0 +1,73 @@
+import './composition-api'; //Due to @vue/composition-api shortcomings, add plugin prior to use in kolibri, studio and tests
+import { onBeforeUnmount, onMounted, ref } from '@vue/composition-api';
+
+/** Window dimensions */
+const windowWidth = ref(null);
+const windowHeight = ref(null);
+
+/**
+ * Update window width and height
+ */
+const updateWindow = () => {
+  const metrics = windowMetrics();
+  windowWidth.value = metrics.width;
+  windowHeight.value = metrics.height;
+};
+
+/**
+ * Returns window inner width and height
+ * @returns {Object} The window's inner width and height
+ */
+function windowMetrics() {
+  return {
+    width: window.innerWidth,
+    height: window.innerHeight,
+  };
+}
+
+/**
+ * Check if Nuxt is server side rendering
+ * @returns {Boolean}
+ */
+function isNuxtServerSideRendering() {
+  return process && process.server;
+}
+
+/**
+ * Add window listener
```

Comment:
Non-blocking just some thoughts on style.

On some of the comments which describe a function like "addWindowListener" with "Add window listener" -- I think it's okay to just leave off the "Add window listener" bit altogether since it's not giving any info the function name doesn't already give. I think it would be helpful in this particular case to share some more context (ie, why do we do two different events depending on whether we're in Nuxt or not?).

On the other hand, the comment description for `removeWindowListener` seems unnecessary.

I love the JSDoc `@param` and type declarations throughout <3